acos_client/v30/slb/service_group.py

```diff
@@ -64,7 +64,13 @@ def _set(self, name, protocol=None, lb_method=None, hm_name=None,
             })
         }
 
-        health_check_disable = 1 if kwargs["health_check_disable"] is True else 0
+        health_check_disable_key = "health_check_disable"
+        health_check_disable = 1
+
+        # If we explicitly disable health checks, ensure it happens
+        # Else, we implicitly disable health checks if not specified.
+        if health_check_disable_key in kwargs:
+            health_check_disable = 1 if kwargs[health_check_disable_key] is True else 0
```

Comment:
Ok, the whole thing above can be:

health_check_disable = 1 if kwargs.get('health_check_disable', False) else 0